### PR TITLE
Update username allowed chars regex

### PR DESCRIPTION
**Pull request by luisrudge**
fix https://github.com/auth0/lock/issues/1072

### CODE REVIEW

**Comment by luisrudge** [comment]
For the future: the translation code needs async/await support, so you have to use nodejs 7.6+

File: src/i18n/kr.js [hunk 1]
```diff
@@ -16,10 +19,11 @@ export default {
       'lock.unauthorized': '권한이 없습니다. 다시 시도해 주세요.',
       'lock.mfa_registration_required': '다단계 인증이 필요하지만, 디바이스가 등록되어 있지 않습니다. 먼저 디바이스를 등록하여 주세요.',
       'lock.mfa_invalid_code': '유효하지 않은 코드입니다. 다시 시도해 주세요.',
-      password_change_required: '처음 로그인하거나, 또는 비밀번호의 기한이 만료되어 비밀번호를 갱신해야 합니다.', // TODO: verify error code
+      password_change_required: '처음 로그인하거나, 또는 비밀번호의 기한이 만료되어 비밀번호를 갱신해야 합니다.',
       password_leaked: '본 계정은 외부 접근으로 인한 회원 정보 유출이 우려되어, 보안 상의 문제로 비활성화되었습니다. 계정을 다시 활성화하는 방법은 자동 전송된 이메일에 안내되어 있습니다.',
       too_many_attempts: '본 계정은 단시간에 복수의 로그인 시도가 감지되어 차단되었습니다.',
-      session_missing: '인증 요청이 완료되지 않았습니다. 열려있는 모든 다이얼로그 창을 닫고 다시 시도해 주시기 바랍니다.'
+      session_missing: '인증 요청이 완료되지 않았습니다. 열려있는 모든 다이얼로그 창을 닫고 다시 시도해 주시기 바랍니다.',
+      'hrd.not_matching_email': 'Please, use your corporate email to login.'
```
Comment (lbalmaceda) [inline src/i18n/kr.js]: This one is not korean 👎. The key seems to have quotes `'` and that's somehow breaking the script?

File: src/i18n/kr.js [hunk 2]
```diff
@@ -98,7 +101,7 @@ export default {
   ssoEnabled: '통합 인증이 유효합니다.',
   submitLabel: '전송',
   unrecoverableError: '오류가 발생하였습니다.<br />고객센터로 연락하여 주시기 바랍니다.',
-  usernameFormatErrorHint: '%d-%d의 문자, 숫자, "_"이/가 사용 가능합니다.',
+  usernameFormatErrorHint: 'Use %d-%d letters, numbers and the following characters: "_", ".", "+", "-"',
```
Comment (lbalmaceda) [inline src/i18n/kr.js]: Not korean

File: src/i18n/kr.js [hunk 3]
```diff
@@ -109,5 +112,6 @@ export default {
   mfaLoginTitle: '2단계 인증',
   mfaLoginInstructions: '스마트폰 어플리케이션으로 생성된 확인 코드를 입력하여 주세요.',
   mfaSubmitLabel: '로그인',
-  mfaCodeErrorHint: '%d 숫자를 사용하여 주세요.'
+  mfaCodeErrorHint: '%d 숫자를 사용하여 주세요.',
+  showPassword: 'Show password'
```
Comment (lbalmaceda) [inline src/i18n/kr.js]: not Korean

File: src/i18n/pt-br.js [hunk 4]
```diff
@@ -96,7 +96,7 @@ export default {
   ssoEnabled: 'Single Sign-On habilitado',
   submitLabel: 'Enviar',
   unrecoverableError: 'Algo deu errado.<br />Por favor entre em contato com o suporte.',
-  usernameFormatErrorHint: 'Use %d-%d letras, números e "_"',
+  usernameFormatErrorHint: 'Use %d-%d letras, números e os seguintes caracteres: "_", ".", "+", "-"',
```
Comment (lbalmaceda) [inline src/i18n/pt-br.js]: Is this br ok? :trollface:

File: src/i18n/kr.js (hunk 1 above)
Comment (luisrudge) [inline src/i18n/kr.js]: the key has quotes because it has a `.` in it, so you have to make it a string. I'll check what's going on.

**Pull request merged**